runelite-client/src/main/java/net/runelite/client/plugins/hiscore/NameAutocompleter.java

```diff
@@ -70,9 +79,11 @@
 	private Pattern autocompleteNamePattern;
 
 	@Inject
-	private NameAutocompleter(@Nullable Client client)
+	private NameAutocompleter(@Nullable Client client, HiscoreConfig config)
 	{
 		this.client = client;
+		this.config = config;
+		searchHistory.addAll(Text.fromCSV(config.searchHistory()));
```

Comment:
This will need to be able to populate when you log into a runelite account too

Reply:
Irrelevant now with persistence functionality removed